### PR TITLE
New algorithim

### PR DESCRIPTION
**Pull request by erdum**
First time contribution

### CODE REVIEW

**Comment by erdum** [comment]
What is the problem i did not understand there is no issue with my file but with someone's else, so what i can do for to pass checks????

**Comment by Lugixion** [comment]
That's a problem with other files so don't worry about it.

File: Electronics_Algorithms/Ohms_law.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def ohms_law(v=0, i=0, r=0):
```
Comment (bupboi1337) [inline Electronics_Algorithms/Ohms_law.py:1]: Add comments for more readability

**Review by bupboi1337 — COMMENTED** [review]
All this needs is some more comments.

**Comment by bupboi1337** [comment]
man, 1 year

**Pull request closed without merging**